flink-runtime/src/test/java/org/apache/flink/runtime/clusterframework/BootstrapToolsTest.java

```diff
@@ -590,75 +595,79 @@ public void testGetDynamicPropertiesAsString() {
+            assertThat(dynamicProperties).isEqualTo("-Dkey.b='b2' -Dkey.c='c'");
         }
     }
 
     @Test
-    public void testEscapeDynamicPropertyValueWithSingleQuote() {
+    void testEscapeDynamicPropertyValueWithSingleQuote() {
         final String value1 = "#a,b&c^d*e@f(g!h";
-        assertEquals("'" + value1 + "'", BootstrapTools.escapeWithSingleQuote(value1));
+        assertThat(BootstrapTools.escapeWithSingleQuote(value1)).isEqualTo("'" + value1 + "'");
 
         final String value2 = "'foobar";
-        assertEquals("''\\''foobar'", BootstrapTools.escapeWithSingleQuote(value2));
+        assertThat(BootstrapTools.escapeWithSingleQuote(value2)).isEqualTo("''\\''foobar'");
 
         final String value3 = "foo''bar";
-        assertEquals("'foo'\\'''\\''bar'", BootstrapTools.escapeWithSingleQuote(value3));
+        assertThat(BootstrapTools.escapeWithSingleQuote(value3)).isEqualTo("'foo'\\'''\\''bar'");
 
         final String value4 = "'foo' 'bar'";
-        assertEquals("''\\''foo'\\'' '\\''bar'\\'''", BootstrapTools.escapeWithSingleQuote(value4));
+        assertThat(BootstrapTools.escapeWithSingleQuote(value4))
+                .isEqualTo("''\\''foo'\\'' '\\''bar'\\'''");
     }
 
     @Test
-    public void testEscapeDynamicPropertyValueWithDoubleQuote() {
+    void testEscapeDynamicPropertyValueWithDoubleQuote() {
         final String value1 = "#a,b&c^d*e@f(g!h";
-        assertEquals("\"#a,b&c\"^^\"d*e@f(g!h\"", BootstrapTools.escapeWithDoubleQuote(value1));
+        assertThat(BootstrapTools.escapeWithDoubleQuote(value1))
+                .isEqualTo("\"#a,b&c\"^^\"d*e@f(g!h\"");
 
         final String value2 = "foo\"bar'";
-        assertEquals("\"foo\\\"bar'\"", BootstrapTools.escapeWithDoubleQuote(value2));
+        assertThat(BootstrapTools.escapeWithDoubleQuote(value2)).isEqualTo("\"foo\\\"bar'\"");
 
         final String value3 = "\"foo\" \"bar\"";
-        assertEquals("\"\\\"foo\\\" \\\"bar\\\"\"", BootstrapTools.escapeWithDoubleQuote(value3));
+        assertThat(BootstrapTools.escapeWithDoubleQuote(value3))
+                .isEqualTo("\"\\\"foo\\\" \\\"bar\\\"\"");
     }
 
     @Test
-    public void testGetEnvironmentVariables() {
+    void testGetEnvironmentVariables() {
         Configuration testConf = new Configuration();
         testConf.setString("containerized.master.env.LD_LIBRARY_PATH", "/usr/lib/native");
 
         Map<String, String> res =
                 ConfigurationUtils.getPrefixedKeyValuePairs("containerized.master.env.", testConf);
 
-        Assert.assertEquals(1, res.size());
+        assertThat(res).hasSize(1);
         Map.Entry<String, String> entry = res.entrySet().iterator().next();
-        Assert.assertEquals("LD_LIBRARY_PATH", entry.getKey());
-        Assert.assertEquals("/usr/lib/native", entry.getValue());
+        assertThat(entry.getKey()).isEqualTo("LD_LIBRARY_PATH");
+        assertThat(entry.getValue()).isEqualTo("/usr/lib/native");
```

Comment:
Maybe it can be simplified as
```java
assertThat(res).hasSize(1).containsEntry("LD_LIBRARY_PATH", "/usr/lib/native")
```

Reply:
Good suggestion, it's very concise! updated